### DISCUSSION TITLE
Materialized tests in dbt core - yay or nay?

### DISCUSSION THREAD

**Discussion by Fleid** [MaterializeInc/materialize#17571] [Technical musings]
I try to @ you from the [issue](https://github.com/dbt-labs/dbt-core/issues/6914) on our repo but couldn't...
So here I am @ahelium and @morsapaes ;)

**Comment by morsapaes** [comment]
Thanks, @Fleid! Going over the recent materialized view musings in the dbt repo and leaving comments as I go.
